kotlin/codegen/src/main/java/com/squareup/moshi/kotlin/codegen/JsonClassCodegenProcessor.kt

```diff
@@ -134,7 +145,16 @@ public class JsonClassCodegenProcessor : AbstractProcessor() {
           }
 
         preparedAdapter.spec.writeTo(filer)
-        preparedAdapter.proguardConfig?.writeTo(filer, type)
+        if (generatedProguard) {
+          preparedAdapter.proguardConfig?.writeTo(filer, type)
+        } else {
+          messager.printMessage(
```

Comment:
Let's not print a noisy warning for this and trust that people specifying the option to disable it know what they're doing 👍

Reply:
removed printing